src/cpp/rtps/messages/RTPSMessageGroup.cpp

```diff
@@ -17,18 +17,22 @@
  *
  */
 
-#include <fastdds/rtps/messages/RTPSMessageGroup.h>
+#include <algorithm>
+
+#include <rtps/flowcontrol/FlowController.h>
+
+#include <rtps/participant/RTPSParticipantImpl.h>
+
+#include <rtps/messages/RTPSGapBuilder.hpp>
+#include <rtps/messages/RTPSMessageGroup_t.hpp>
```

Comment:
Being private headers, they should be the last in the list. Also, I would not leave any blank line between the library headers. Only a blank line separating the public from the private headers.
This criterion should be followed also in the other files that have been modified.